gravitee-apim-console-webui/src/management/api/import-v4/api-import-v4.component.ts

```diff
@@ -69,6 +85,7 @@ export class ApiImportV4Component implements AfterViewInit {
     {
       format: new FormControl('gravitee', [Validators.required]),
       source: new FormControl('local', [Validators.required]),
+      withDocumentation: new FormControl(false, [Validators.required]),
```

Comment:
The value is not consistent with openApi 🤔  No ? 
If this is the default value, we need to change 
![image](https://github.com/gravitee-io/gravitee-api-management/assets/4974420/2abe1838-9d8e-4b16-8b9c-6241e58eaa71)
or remove default property 
otherwise it will be true if undefined, which may not be desired 💭

Reply:
I hadn't seen the observable that forces a true if !== 'openapi'